src/shared.js

```diff
@@ -2585,13 +2585,14 @@ export const translateNumToType = {
 
 export const convertISODateTime = (isoDateTime) => {
     const date = new Date(isoDateTime);
-    return setZeroDateTime(date.getMonth() + 1)+ '/' + setZeroDateTime(date.getDate()) + '/' + date.getFullYear()+ ' '+ date.getHours() + ':' + setZeroDateTime(date.getMinutes())
-}
+    return setZeroDateTime(date.getUTCMonth() + 1) + '/' + setZeroDateTime(date.getUTCDate()) + '/' + date.getUTCFullYear() + ' ' + setZeroDateTime(date.getUTCHours()) + ':' + setZeroDateTime(date.getUTCMinutes())
+};
+
+// append 0 before min. if single digit min. or hour
+const setZeroDateTime = (dateTimeInput) => {
+    return dateTimeInput < 10 ? '0' + dateTimeInput : dateTimeInput.toString();
+};
```

Comment:
```suggestion
    return date.toLocaleString('en-US');
```

Reply:
Is this (changed date) caused by 'T00:00:00.000Z' appended to the date string?

Reply:
Then we may not need this extra string and extra handlings designed for it, right?
This (clean up 'T00:00:00.000Z') can be on ToDo list.